content/blog/2024/11/04/2024-11-04-jenkins-tutorial-revamp.adoc

```diff
@@ -0,0 +1,273 @@
+This was particularly important for users with less powerful machines or those in environments with limited resources.
+
+3. *Docker Compose profiles*: We introduced profiles for different languages and technologies, making it easy to switch between setups.
+For example, users can start a Maven-specific setup with `docker compose --profile maven up -d`.
+
+4. *Automated updates*: We implemented automation to keep our Docker images up-to-date, using tools like Dependabot and UpdateCLI.
+
+== The Result: A More Inclusive Jenkins
+
+The outcome of our efforts is a Jenkins experience that's more inclusive than ever.
+Let's break down the key improvements:
+
+=== 1. Simplified Setup
+
+Users can now get Jenkins running with a single command. Here's how straightforward it is:
+
+1. Clone the repository: `git clone https://github.com/jenkins-docs/quickstart-tutorials.git`
+2. Navigate to the directory: `cd quickstart-tutorials`
+3. Start Jenkins: `docker compose --profile <tutorial_name> up -d`
+
+That's it! No more complex Docker commands or manual configuration.
+
+=== 2. Browser-based Access with GitPod
+
+Thanks to GitPod integration, Jenkins is now just a click away, even on less powerful devices.
+This cloud-based approach brings several key advantages for accessibility and ease of use:
+
+. _*One-Click Setup*_: The repository's `README` contains a link:https://gitpod.io/#https://github.com/jenkins-docs/quickstart-tutorials[GitPod link] that launches Jenkins instantly.
+With the GitPod Chrome extension installed, you'll see a prominent green button that makes the process even smoother.
+. _*Resource-Friendly*_: GitPod provides 50 hours of free monthly usage, making Jenkins accessible to users regardless of their local hardware capabilities.
+. _*Bandwidth Optimization*_: Since GitPod handles the heavy lifting of downloading Docker images, users can get started quickly even with limited internet connectivity - whether they're working from a train, a remote location, or a region with restricted bandwidth.
+. _*Familiar Development Environment*_: Users get instant access to a Visual Studio Code-like IDE and terminal, complete with clear instructions for their chosen tutorial.
+
+Here's a quick demonstration of the GitPod integration in action:
+
+video::WsH7yL5chow[youtube,width=839]
+
+As you can see, users have some instructions, and a command to copy-paste in the terminal.
+
+After pasting the command in the terminal, the setup process begins automatically.
+Docker images are pulled efficiently through GitPod's infrastructure, preserving your local bandwidth.
+The Jenkins controller and agent containers are launched, and a browser tab automatically opens to display the Jenkins interface.
+The entire process takes less than a minute - a significant improvement over traditional setup methods.
+Since we've streamlined the authentication process, you can log in immediately using the credentials admin/admin and begin your Jenkins journey.
```

Comment:
```suggestion
Since we've streamlined the authentication process, you can log in immediately using the credentials (`admin`/`admin`) and begin your Jenkins journey.
```
Suggestion to update the credentials so they match how they are shown in line 199